client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -28,6 +29,8 @@ object DiagramView {
     // the element() method
     private val chart: ChartView.ChartView = ChartView.ChartView()
 
+    val zipVar = Var("")
```

Comment:
Make the var private ? - Also I am not sure if zipVar is the most natural name for it. perhaps `titleVar` for now ? - Also, shouldn´t it be prepopulated with the current tite ?

Reply:
I made it private, and yes it gets populated as is hooked to the app state. So you will not see it empty in the UI